### ISSUE TITLE
"Error: Invalid Semantic Version" running `make run/pipecd`

### ISSUE THREAD

**Issue by dgannon991** [pipe-cd/pipecd#4845]
**What happened**: Attempting to install pipeCD in my local cluster, it threw an error about invalid semantic versioning

**What you expected to happen**: It installed successfully.

**How to reproduce it**: Follow the local running steps on a linux laptop. 

**Environment**:   
- `piped` version: https://github.com/pipe-cd/pipecd/commit/13c3ca06b6669af999750636091233005b967338
- `control-plane` version: N/A
- Others: 
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"

Helm: version.BuildInfo{Version:"v3.14.3", GitCommit:"f03cc04caaa8f6d7c3e67cf918929150cf6f3f12", GitTreeState:"clean", GoVersion:"go1.21.7"}

**Comment by ffjlabo** [comment]
@dgannon991 Thank you for submit this bug noticed at the contribute fest's one :)
If possible, can I add the logs to the comment?

**Comment by ffjlabo** [comment]
📝 It happens when executing `make run/pipecd`

**Comment by dgannon991** [comment]
Hi Guys,

The logs are

`david@Davids-Laptop:~/Projects/contribfest/pipecd-pipecd$ make run/pipecd
Building go binary of Control Plane...
GOOS=linux GOARCH=amd64 CGO_ENABLED=0  go build -ldflags "-X github.com/pipe-cd/pipecd/pkg/version.version="db771fe-1711281878" -X github.com/pipe-cd/pipecd/pkg/version.gitCommit=db771fe61f5f68d944869b7624759a2c91bdbf68 -X github.com/pipe-cd/pipecd/pkg/version.buildDate=20240324-120438 -w" -o ./.artifacts/pipecd ./cmd/pipecd
Building web static files...
yarn --cwd web build
yarn run v1.22.22
$ PIPECD_VERSION=`git describe --tags --always --dirty --abbrev=7` webpack build --mode production --env htmlTemplate=./base.html --config ./webpack.config.js
assets by status 4.38 MiB [cached] 14 assets
assets by path . 15.7 KiB
  asset favicon.ico 15 KiB [compared for emit]
  asset index.html 635 bytes [compared for emit]
Entrypoint index [big] 1.53 MiB (4.24 KiB) = assets/951.528cb1bdf.js 270 KiB assets/814.e1312c730.js 984 KiB assets/index.e79098ecf.js 318 KiB 1 auxiliary asset
orphan modules 8.46 MiB [orphan] 6913 modules
runtime modules 7.86 KiB 13 modules
cacheable modules 9.39 MiB (javascript) 4.24 KiB (asset) 2552 modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  assets/index.e79098ecf.js (318 KiB)
  assets/951.528cb1bdf.js (270 KiB)
  assets/767.7a1667348.chunk.js (1.92 MiB)
  assets/814.e1312c730.js (984 KiB)
  assets/648.81af83f31.chunk.js (622 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  index (1.53 MiB)
      assets/951.528cb1bdf.js
      assets/814.e1312c730.js
      assets/index.e79098ecf.js


webpack 5.76.0 compiled with 2 warnings in 14373 ms
Done in 15.23s.
Building docker image and pushing it to local registry...
docker build -f cmd/pipecd/Dockerfile -t localhost:5001/pipecd:"db771fe-1711281878" .
[+] Building 0.9s (10/10) FINISHED                                                                                                                                                                  docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                                          0.0s
 => => transferring dockerfile: 673B                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/alpine:3.13                                                                                                                                                0.5s
 => [internal] load .dockerignore                                                                                                                                                                             0.0s
 => => transferring context: 2B                                                                                                                                                                               0.0s
 => [1/5] FROM docker.io/library/alpine:3.13@sha256:469b6e04ee185740477efa44ed5bdd64a07bbdd6c7e5f5d169e540889597b911                                                                                          0.0s
 => [internal] load build context                                                                                                                                                                             0.1s
 => => transferring context: 37.72MB                                                                                                                                                                          0.1s
 => CACHED [2/5] RUN     apk add --no-cache         python3         curl &&     curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-324.0.0-linux-x86.tar.gz &&     tar -zxv  0.0s
 => CACHED [3/5] RUN mkdir -p web/static                                                                                                                                                                      0.0s
 => [4/5] ADD .artifacts/pipecd /usr/local/bin/pipecd                                                                                                                                                         0.1s
 => [5/5] ADD .artifacts/web-static web/static                                                                                                                                                                0.0s
 => exporting to image                                                                                                                                                                                        0.1s
 => => exporting layers                                                                                                                                                                                       0.1s
 => => writing image sha256:c5a67bd8d457ebd30cd3edd9af8f16ce12a1735efdf8c72fd31583e6977eca06                                                                                                                  0.0s
 => => naming to localhost:5001/pipecd:db771fe-1711281878                                                                                                                                                     0.0s
docker push localhost:5001/pipecd:"db771fe-1711281878"
The push refers to repository [localhost:5001/pipecd]
221f4e91b83d: Layer already exists 
bba438c2dfb2: Pushed 
448fc077177f: Layer already exists 
11e070ec3584: Layer already exists 
7df5bd7bd262: Layer already exists 
db771fe-1711281878: digest: sha256:455fba718eb712c7601ded86853128be078279ca23e39b26f2febd3382240f67 size: 1371
Installing Control Plane in kind...
mkdir -p .artifacts
helm package manifests/pipecd --version "db771fe-1711281878" --app-version "db771fe-1711281878" --dependency-update --destination .artifacts
Error: Invalid Semantic Version
make: *** [Makefile:116: run/pipecd] Error 1`

It looks like the issue is helm doesn't accept a version number like "db771fe-1711281878". When we hardcoded this to 0.0.1 it worked just fine :) 

Cheers
